build.gradle.kts

```diff
@@ -0,0 +1,193 @@
+plugins {
+    java
+}
+
+repositories {
+    mavenCentral()
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+}
```

Comment:
Is this version a minimum requirement for JUnit 5?

Reply:
@anirudha13 - the JUnit5 recommendation is Java 8 and we already have that in the README.md. i.e. Java 8+

I referred this link below while keeping Java 11 as default: https://www.oracle.com/java/technologies/java-se-support-roadmap.html
Java 8 goes out of premier support in 1 year whereas Java 11 has 2 years to go.

Your suggestion for default?

Reply:
I would say stick to Java 8 for now, since that will communicate to customers that this code still supports Java 8.

Reply:
Changed to Java 8

Reply:
this has been done, changed to java 8 from java 11